cli/azd/pkg/project/framework_service_docker.go

```diff
@@ -32,6 +26,11 @@ import (
 	"github.com/azure/azure-dev/cli/azd/pkg/tools/docker"
 	"github.com/azure/azure-dev/cli/azd/pkg/tools/pack"
 	"go.opentelemetry.io/otel/trace"
+	"log"
+	"os"
+	"path"
+	"path/filepath"
+	"strings"
```

Comment:
revert this change. Don't add unnecessary changes.

You can configure your Go Land like this:

> ![image](https://github.com/user-attachments/assets/b0a69fc5-5dd2-4a27-bf82-a78ef81f8357)

Reply:
Updated, thanks!